core/src/main/java/com/cloud/agent/api/MigrateCommand.java

```diff
@@ -40,6 +40,7 @@
     private boolean executeInSequence = false;
     private List<MigrateDiskInfo> migrateDiskInfoList = new ArrayList<>();
     private Map<String, DpdkTO> dpdkInterfaceMapping = new HashMap<>();
+    Map<String, Boolean> vlanToPersistenceMap;
```

Comment:
Just a note - if field is not initialised, the consumer of getter must do a NPE check.